docs/entities/CaseDeadline.md

```diff
@@ -75,5 +89,10 @@
             limit: 1
       allow: 
         - "CaseDeadline"
+    sortableDocketNumber: 
+      type: "number"
+      flags: 
+        presence: "required"
+        description: "A sortable representation of the docket number of the case containing the Case Deadline."
```

Comment:
Is sortableDocketNumber required on migrated Case Deadlines? If so, does the `sortable representation` need to match the representation used internally by ef-cms? If that's the case, then this format should be documented here. Alternatively, since the sortable representation is an internal detail of how ef-cms works, this could be generated by from the passed-in docket number. This would be preferable so the migration code would not be required to know private implementation details of ef-cms and will not break if that representation changes.